### PR TITLE
Tri ngo

### PR DESCRIPTION
**Pull request by Keido0208**

### CODE REVIEW

File: .flutter-plugins
Comment (Kenneth-W-Chen) [inline .flutter-plugins:1]: Remove this from git

File: .flutter-plugins-dependencies
Comment (Kenneth-W-Chen) [inline .flutter-plugins-dependencies:1]: Remove this from git

File: pubspec.lock
Comment (Kenneth-W-Chen) [inline pubspec.lock:1]: You deleted `sqflite`. Put it back

**Review by Kenneth-W-Chen — CHANGES_REQUESTED** [review]
Remove the entire `backend` folder. Only push what's finished and what we're working on for this sprint
Remove `.dart_tool`

File: pubspec.lock
Comment (Keido0208) [reply pubspec.lock:1]: Put bacj sqflite

**Pull request closed without merging**